eng/scripts/Get-ApiViewCommandLine.ps1

```diff
@@ -50,4 +50,6 @@ foreach ($incDir in $AllLibIncDirs) {
     $CmdLine += " -I " + $incDir
 }
 
+$CmdLine += " -D_azure_APIVIEW"
```

Comment:
I'd recommend adding doc comments in the source to describe why we need this (similar to your explanation in the PR description).